pkg/storage/streams/store.go

```diff
@@ -59,15 +59,16 @@ type Store interface {
 
 // streamStore is a store for streams
 type streamStore struct {
-	segments     segments.Store
-	segmentSize  int64
-	rootKey      *storj.Key
-	encBlockSize int
-	cipher       storj.Cipher
+	segments        segments.Store
+	segmentSize     int64
+	rootKey         *storj.Key
+	encBlockSize    int
+	cipher          storj.Cipher
+	inlineThreshold int
 }
 
 // NewStreamStore stuff
-func NewStreamStore(segments segments.Store, segmentSize int64, rootKey *storj.Key, encBlockSize int, cipher storj.Cipher) (Store, error) {
+func NewStreamStore(segments segments.Store, segmentSize int64, rootKey *storj.Key, encBlockSize int, cipher storj.Cipher, inlineThreshold int) (Store, error) {
```

Comment:
Note: the number of arguments is getting large here.

Reply:
True, the list is growing. But we are going to replace this code with a brand new one around the Metainfo Refactoring, right?

Reply:
Yeah, eventually.